### PR TITLE
update go client to use latest openapi spec

### PR DESCRIPTION
**Pull request by pmenglund**
updated the openapi spec to the latest published version and made all tests pass

### CODE REVIEW

File: openapi/api/openapi.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ info:
   title: REST API
   version: v1
 servers:
-- url: https://api.rs2.usw2.rockset.com/
+- url: https://api.use1a1.rockset.com/
```
Comment (adamsb6) [inline openapi/api/openapi.yaml:17]: Is this right? Why not usw2a1?

File: openapi/docs/SourceSnowflake.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+and makes sure properties required by API are set, but the set of arguments
+will change when the set of required properties is changed
+
+### NewSourceSnowflakeWithDefaults
+
+`func NewSourceSnowflakeWithDefaults() *SourceSnowflake`
+
+NewSourceSnowflakeWithDefaults instantiates a new SourceSnowflake object
+This constructor will only assign default values to properties that have it defined,
+but it doesn't guarantee that properties required by API are set
```
Comment (adamsb6) [inline openapi/docs/SourceSnowflake.md:30]: These lack an ending period, but all others seem to have one.

File: openapi/docs/Stats.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+and makes sure properties required by API are set, but the set of arguments
+will change when the set of required properties is changed
+
+### NewStatsWithDefaults
+
+`func NewStatsWithDefaults() *Stats`
+
+NewStatsWithDefaults instantiates a new Stats object
+This constructor will only assign default values to properties that have it defined,
+but it doesn't guarantee that properties required by API are set
```
Comment (adamsb6) [inline openapi/docs/Stats.md:29]: No ending period.

File: openapi/docs/StatusSnowflake.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+and makes sure properties required by API are set, but the set of arguments
+will change when the set of required properties is changed
+
+### NewStatusSnowflakeWithDefaults
+
+`func NewStatusSnowflakeWithDefaults() *StatusSnowflake`
+
+NewStatusSnowflakeWithDefaults instantiates a new StatusSnowflake object
+This constructor will only assign default values to properties that have it defined,
+but it doesn't guarantee that properties required by API are set
```
Comment (adamsb6) [inline openapi/docs/StatusSnowflake.md:26]: No ending period.

**Review by adamsb6 — APPROVED** [review]
I assume all the  openapi/model files are generated, and I gave the docs a light skim. Only noticed an inconsistency in how sometimes we end a sentence with a period and sometimes do not.

File: openapi/api/openapi.yaml (hunk 1 above)
Comment (pmenglund) [reply openapi/api/openapi.yaml:17]: I've got a linear issue to fix that, this diff is just an openapi update

File: openapi/docs/Stats.md (hunk 3 above)
Comment (pmenglund) [reply openapi/docs/Stats.md:29]: this is auto-generated

File: openapi/docs/StatusSnowflake.md (hunk 4 above)
Comment (pmenglund) [reply openapi/docs/StatusSnowflake.md:26]: this is auto-generated

File: openapi/docs/SourceSnowflake.md (hunk 2 above)
Comment (pmenglund) [reply openapi/docs/SourceSnowflake.md:30]: this is auto-generated

**Pull request merged**